### ISSUE TITLE
Add to CREDITS.md

### ISSUE THREAD

**Issue by arshadkazmi42** [arshadkazmi42/arshadkazmi42.github.io#17]
Add this url with the thank you note in [CREDITS.md](https://github.com/arshadkazmi42/arshadkazmi42.github.io/blob/develop/CREDITS.md)

`https://dribbble.com/shots/2316666-Profile-card-UI`

This is designer's profile 
https://dribbble.com/sokratus

You can get name of the designer also from the above url.

UI of this application is inspired from this.

**Comment by Killernotch101** [comment]
I can do it!

**Comment by arshadkazmi42** [comment]
@Killernotch101 Thanks. Go ahead and create a PR

**Comment by Killernotch101** [comment]
Is there any license?

**Comment by arshadkazmi42** [comment]
No. We just want to mention the designer name and profile with link. of the UI design in CREDITS file and it will be great to have a thank you note in credit file
